acceptance/bundle/environments/dependencies/output.txt

```diff
@@ -0,0 +1,37 @@
+
+>>> [CLI] bundle validate
+Name: dependencies
+Target: default
+Workspace:
+  User: [USERNAME]
+  Path: /Workspace/Users/[USERNAME]/.bundle/dependencies/default
+
+Validation OK!
```

Comment:
Would be nice to see `bundle validate -o json | jq . resources.jobs.test_job.environments[0].spec.dependencies`

Reply:
Added here https://github.com/databricks/cli/pull/2736/commits/ee64e50560029a499a95fb4a4646c3a6811bdcae